src/components/Pickers/Lookup/index.d.ts

```diff
@@ -1,9 +1,15 @@
 import * as React from 'react';
 
-export interface LookupValue {
-	value?: any;
-	label?: string;
-}
+export type LookupValue =
+	| {
```

Comment:
Why not define using the interface above?

Reply:
Typings are autogenerated from PropTypes. Better typings will appear when we move to typescript.